sycl/test-e2e/Graph/exception_unsupported_backend.cpp

```diff
@@ -0,0 +1,34 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// Tests the ability to finalize a empty command graph
+// without submitting the graph.
```

Comment:
Description looks like a copy-paste, should mention checking an exception is thrown when creating a graph for an unsupported backend